### PR TITLE
Switch to ubi9/ubi-minimal from centos/stream 8 & 9

### PR DESCRIPTION
**Pull request by deepsm007**
Follow-up of https://github.com/openshift/release/pull/52822 

/cc @openshift/test-platform @danilo-gemoli

### CODE REVIEW

**Comment by danilo-gemoli** [comment]
https://github.com/openshift/release/pull/53426 should fix the problem

**Comment by deepsm007** [comment]
https://github.com/openshift/release/pull/53437 will fix the final failure

File: images/job-run-aggregator/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/centos/centos:stream9
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 LABEL maintainer="deads@redhat.com"
```
Comment (danilo-gemoli) [inline images/job-run-aggregator/Dockerfile:2]: Small issue here, you forgot to use `microdnf`

Comment (deepsm007) [reply images/job-run-aggregator/Dockerfile:2]: Looks like added recently after rebase. Updated.

File: images/autoperibolossync/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,10 @@
-FROM quay.io/centos/centos:stream9
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 
-ADD private-org-peribolos-sync /usr/bin/private-org-peribolos-sync
-ADD autoperibolossync /usr/bin/autoperibolossync
+#ADD private-org-peribolos-sync /usr/bin/private-org-peribolos-sync
```
Comment (danilo-gemoli) [inline images/autoperibolossync/Dockerfile:3]: Why have these two lines been commented?

File: images/ci-operator-minimal/Dockerfile [hunk 3]
```diff
@@ -1,8 +1,5 @@
 FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 
-RUN microdnf install -y git python3
```
Comment (danilo-gemoli) [inline images/ci-operator-minimal/Dockerfile:3]: Isn't python needed anymore?

Comment (deepsm007) [reply images/ci-operator-minimal/Dockerfile:3]: This particular dockerfile is not getting used by any jobs. I am updating this image to test without python from an earlier discussion on thread between Brad and Danilo.

File: images/autoperibolossync/Dockerfile (hunk 2 above)
Comment (deepsm007) [reply images/autoperibolossync/Dockerfile:3]: These were for testing the image, let me fix this.

File: images/ci-operator-minimal/Dockerfile (hunk 3 above)
Comment (danilo-gemoli) [reply images/ci-operator-minimal/Dockerfile:3]: Alright got it, this is `ci-operator-minimal`, we also have the fully fledged `ci-operator`.

**Comment by danilo-gemoli** [comment]
`ci-operator` image should be fine even without `RUN alternatives --install /usr/bin/python python /usr/bin/python3.9 1`:
```sh
$ b01 get -f pod.yaml -oyaml | yq .spec.containers[].image
registry.ci.openshift.org/ocp/ubi-minimal:9
$ b01 --as system:admin rsh -f pod.yaml
sh-5.1# microdnf install -y git python3
...
Complete.
sh-5.1# python -c 'print("hello")'
hello
sh-5.1# python --version
Python 3.9.18
sh-5.1# ls -l /usr/bin/python
lrwxrwxrwx. 1 root root 9 Jan 29 09:08 /usr/bin/python -> ./python3
```

**Pull request merged**